src/main/java/com/amihaiemil/charles/github/Language.java

```diff
@@ -26,21 +26,23 @@
 
 package com.amihaiemil.charles.github;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.io.IOException;
 import java.util.Properties;
 import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 /**
  * Language that the agent speaks.
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
  * @since 1.0.0
- * 
+ *
+ * @todo #174:30m/DEV Add commands_ru.properties files for commands and responses
```

Comment:
@salikjan You can remove the empty line above, the todo can be just under ``@since``.
And can you add also this todo?

```
@todo #174:30m/DEV Implement and unit test Language class Russian.
 Follow class English and its unit tests as example.
```

And, I don't think you commited the changes to ``.gitignore``, since it doesn't appear here as changed file :D